simulator/trainingbot_agent.py

```diff
@@ -84,4 +104,86 @@ def update(self):
           aspect=1.0,
           nearVal=0.1,
           farVal=3.1)
-        p.getCameraImage(300, 300, view_matrix, projection_matrix, renderer=p.ER_BULLET_HARDWARE_OPENGL)
+        #p.getCameraImage(300, 300, view_matrix, projection_matrix, renderer=p.ER_BULLET_HARDWARE_OPENGL)
+
+    def generate_target_velocities(self):
+        R = .045
+        L = .1
+        D = .2427
+
+        BEZIERX = [-.93,2,-1,0]
+        BEZIERY = [0,0,1,-2]
+
+        curve = Utilities.add_bezier_curve(BEZIERX, BEZIERY)
+
+        currentTheta = 0.0
+        currentU = 0.0
+
+        lastCtrlPt = p.multiplyTransforms(
+                        p.getBasePositionAndOrientation(self.robot)[0],
+                        p.getBasePositionAndOrientation(self.robot)[1],
+                        (0,0,L + 0.074676),
+                        (0,0,0,1))[0]
+
+        lid = p.addUserDebugLine(
+                        [BEZIERX[0]+BEZIERX[1]*currentU+BEZIERX[2]*currentU**2+BEZIERX[3]*currentU**3,
+                         BEZIERY[0]+BEZIERY[1]*currentU+BEZIERY[2]*currentU**2+BEZIERY[3]*currentU**3,
+                         -10],
+                        [BEZIERX[0]+BEZIERX[1]*currentU+BEZIERX[2]*currentU**2+BEZIERX[3]*currentU**3,
```

Comment:
**TODO:** Move all of this work to [Planning2020](https://github.com/GT-IEEE-Robotics/Planning2020) repo.